packages/contracts/script/Deploy.sh

```diff
@@ -0,0 +1,27 @@
+# Expects jq to be installed
```

Comment:
what do you think about replacing the `contracts/deploy.sh` with this one?

Reply:
I think both are unnecessary : ) 

Do you want to try out the new deploy script a few times, and if it looks good, one of us pushes an update to remove the existing one?

Reply:
Can you share more about why both are unnecessary? The shell script is/was partly there to deploy (now replaced by this solidity script 🙏 ) but also partly there to track the addresses for each contract so that the app, etc. can import them for wiring up contract interactions in the frontend. Is there another approach you're thinking for the latter step?

Reply:
I think I misunderstood the question : )

I thought you referring to having both the deploy.sh file in the project root and also having one in the scripts folder. I think having at least one shell script is still necessary, but having two that do the same thing is not necessary. With anything that requires multiple flags or options to remember, I love have a easy script file to wrap all that up into a single command.